lib/pxcontext.js

```diff
@@ -52,6 +52,11 @@ class PxContext {
                 }
             });
         }
+        if (this.uri.includes('graphql')) {
+            config.logger.debug('Graphql route detected');
+            this.graphqlData = pxUtil.getGraphqlData(req);
+            this.sensitiveGraphqlOperation = this.isSensitiveGraphqlOperation(config, this.graphqlData);
```

Comment:
why pass a member (this) as a parameter?

Reply:
Done